nautobot/circuits/views.py

```diff
@@ -102,6 +103,27 @@ class ProviderUIViewSet(NautobotUIViewSet):
     queryset = Provider.objects.all()
     serializer_class = serializers.ProviderSerializer
     table_class = tables.ProviderTable
+    object_detail_content = ObjectDetailContent(
+        panels=(
+            ObjectFieldsPanel(
+                section=SectionChoices.LEFT_HALF,
+                weight=100,
+                fields="__all__",
+                exclude_fields=["name", "comments"],
+            ),
+            ObjectFieldsPanel(
+                label="Comments",
+                weight=200,
+                section=SectionChoices.RIGHT_HALF,
+                fields=["comments"],
+                value_transforms={"comments": [render_markdown, placeholder]},
+            ),
```

Comment:
Let's look at making `comments` an automatic exclusion and automatic separate panel much like `tags` is at present. Maybe out of scope for this PR; if you'd prefer to open as a separate issue that's fine by me.

Reply:
https://github.com/nautobot/nautobot/issues/6321